### PR TITLE
Reduce baremetal-operator pod-initialization time

### PR DESCRIPTION
**Pull request by kashifest**
This PR pre-pulls images in minikube to reduce the baremetal-operator pod-initialization time.

### CODE REVIEW

File: 01_prepare_host.sh [hunk 1]
```diff
@@ -92,6 +92,12 @@ function init_minikube() {
     #If the vm exists, it has already been initialized
     if [[ "$(sudo virsh list --all)" != *"minikube"* ]]; then
       sudo su -l -c "minikube start" "$USER"
+      # Pre-pull the image to reduce pod initialization time
+      for IMAGE_VAR in IRONIC_IMAGE IPA_DOWNLOADER_IMAGE VBMC_IMAGE SUSHY_TOOLS_IMAGE BAREMETAL_OPERATOR_IMAGE; do
```
Comment (maelk) [inline 01_prepare_host.sh]: VBMC and SUSHY images are not needed

File: 01_prepare_host.sh [hunk 2]
```diff
@@ -92,6 +92,12 @@ function init_minikube() {
...
+        IMAGE=${!IMAGE_VAR}
+        sudo su -l -c "minikube ssh sudo docker pull "$IMAGE"" "${USER}"
```
Comment (maelk) [inline 01_prepare_host.sh]: you do not need the indirect expansion if you use the variables content here directly

File: lib/common.sh [hunk 3]
```diff
@@ -62,10 +62,13 @@ export SUSHY_TOOLS_IMAGE=${SUSHY_TOOLS_IMAGE:-"quay.io/metal3-io/sushy-tools"}
 
 # Ironic vars
 export IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
-export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
```
Comment (maelk) [inline lib/common.sh:65]: why removing ? master and latest are different

**Review by maelk — CHANGES_REQUESTED** [review]

File: 01_prepare_host.sh (hunk 1 above)
Comment (kashifest) [reply 01_prepare_host.sh]: right. I forgot to remove them

File: 01_prepare_host.sh (hunk 2 above)
Comment (kashifest) [reply 01_prepare_host.sh]: Copying the same coding conventions used above

File: lib/common.sh (hunk 3 above)
Comment (kashifest) [reply lib/common.sh:65]: Okay. I thought, the same image could be used since baremetal-operator uses latest

**Pull request merged**